targets/TARGET_NUVOTON/TARGET_M2351/TARGET_M23_NS/TARGET_NU_PFM_M2351_NPSA_NS/TARGET_NU_PREBUILD_SECURE/NuMaker-mbed-TZ-secure-example.hex

```diff
@@ -238,1336 +237,1323 @@
-:1043300040F2C0343C3481002140800840F2C03259
-:104340001040264D2843012200924250214F38460A
-:1043500000F077FF0298411C37D081000194214082
-:10436000800840F2C03210402843002242503846B4
-:1043700000F067FF761C042E24D300270298411C0E
-:1043800023D08100019E3140800840F2C0321040AD
-:10439000124C2043009A42500F4D284600F051FF26
-:1043A0000298411C11D081003140800840F2C03297
-:1043B0001040204300264650284600F042FF7F1C54
-:1043C000042FDBD30298411CB2D1054805492822AD
-:1043D000FFF798FFF0490200801A060000480050DD
-:1043E000AC590000C4590000F0B581B07646009386
-:1043F0000D460121094BD3E84F7FC3E8441FFF213D
-:104400000F4209D1069F11463246FFF77DFD28462F
-:1044100000993A4600F060F900F056F8CC280020E8
-:1044200083B080B581B003AF0EC703A9009100F03F
-:104430002FF801B080BC01BC03B00047B0B582B01A
-:1044400004460220002521462A4600F009FFFFF716
-:10445000DFF911E0E8B20D2806D001A90D2008709F
-:104460000220012200F0FCFE02200122214600F081
-:10447000F7FE601C2578044620780A28EAD0002838
-:10448000F2D102B0B0BC01BC8646FFF703BA00000F
-:1044900010B5A2B00B46024601A88421FCF760F8D3
-:1044A0000446842805D301A87F3005A10522FCF726
-:1044B00009FA012C02DB01A8FFF7C0FF22B010BDF2
-:1044C0002E2E2E0A0000000080B50D480122D0E8F3
-:1044D0004F1FC0E8432F002BF8D1C80705D0FFF7C6
-:1044E00097F9BFF34F8F20BFFDE7FFF7E1F928B938
-:1044F000FFF785F910B10120FFF718FAFFF70EFF5B
-:10450000B429002070470000F0B581B01B48017845
-:1045100091BB012404701A4F59203860162038606E
-:104520008820386038680028F6D00420FDF704FEA3
-:104530002046FDF701FE08252846FDF7FDFD022671
-:104540003046FDF7F9FDA004FDF7F6FD1020FEF75B
-:1045500075F82046FEF772F82846FEF76FF83046E9
-:10456000FEF76CF84020FEF769F80648FDF792FF69
-:10457000FEF79AFA0020386001B0F0BDEC29002067
-:10458000000100400090D00306480021017000F0B7
-:104590005FBDC04680B500F06FFD0248012101708B
-:1045A00080BDC046B828002070B5044600F060FD0C
-:1045B0004068012181404E1E00F05AFD00680A4902
-:1045C00088420CD100F0F6FD054600F0F3FD411BDA
-:1045D0003140621A8C4205461446F6D270BD03A0E3
-:1045E0000DA13822FFF78EFE40420F0075735F74F5
-:1045F00069636B65725F6765745F696E666F2829B2
-:104600002D3E6672657175656E6379203D3D203182
-:1046100030303030303000002E5C6D6265642D6FBC
-:10462000735C746172676574735C5441524745549E
-:104630005F4E55564F544F4E5C6E755F74696D6595
-:10464000722E630038F05CBDB0B5421C1BD042082E
-:104650007C231340420978241440E2180C4BD458B0
-:1046600080001C2505400F20A84084430C43D450F3
-:10467000B0BC01BC86467046714672467346F4462D
-:104680008EF300887447124812496422FFF73AFEFD
-:104690003000004080B5421C12D0C91ECAB2032AA5
-:1046A0000DD8C3B201219940800840F2C0330340C5
-:1046B0000648184380BC08BC9E46FDF7D6BF80BDA7
-:1046C000034804492722FFF71DFEC0460040005062
-:1046D000B1590000F65B0000F0B581B000920D46C4
-:1046E0000646032040073E4C21680A4602404F42DE
-:1046F000824200D08FB2080CC3B23A483A46FFF764
-:104700008FFE39483818132845D8810001A2511866
-:104710008F46C04600F026B800F027B800F028B851
-:1047200000F029B800F02AB800F02BB800F02CB83F
-:1047300000F02DB800F02FB800F02DB800F02BB825
-:1047400000F029B800F027B800F025B8
```

Comment:
What is the maintenance plan for this binary? When will it need to be rebuilt? When will we notice that it is broken and requires rebuilding? Maybe some sort of automated test is needed.

Reply:
@0xc0170 How do we indicate the license this binary is provided under?

Reply:
> What is the maintenance plan for this binary? When will it need to be rebuilt? When will we notice that it is broken and requires rebuilding? Maybe some sort of automated test is needed.

Currently, it is re-built by Nuvoton manually. When there's any of memory re-partition, change of IP security attribute, or change of secure function I/F, it will be re-built.

Reply:
> @0xc0170 How do we indicate the license this binary is provided under?

license file located in the folder